src/js/components/Calendar/utils.js

```diff
@@ -150,7 +158,22 @@ export const normalizeForTimezone = (value, timestamp, normalize = true) => {
   let localOffset = 0;
   if (normalize) {
     if (timestamp && typeof timestamp === 'string') {
+      const day = Array.isArray(value) ? undefined : new Date(value);
+      const today = new Date();
       hourDelta = parseInt(timestamp?.split(':')[0], 10);
+      if (
+        day &&
+        !inDaylightSavings(day) &&
```

Comment:
Do we have to check whether inDaylightSavings(day) !== inDaylightSavings(today)?

Reply:
this line `day.getTimezoneOffset() > today.getTimezoneOffset()` covers that, it one of the days (either day or today) is in daylight savings and the other is not the timezoneOffset will be different. If they are both in or outside of daylight savings the timezoneOffset will be the same

Reply:
Thanks. Can you add that to a comment? We seem to have trouble remembering some of these important details.

Reply:
Yeah I'll add a comment